### ISSUE TITLE
install with a click of a button

### ISSUE THREAD

**Issue by steren** [GoogleCloudPlatform/cloud-run-release-operator#71]
Could the steps at https://github.com/GoogleCloudPlatform/cloud-run-release-operator#setup- be performed after I clicked on an [Run on Google Cloud button](https://github.com/GoogleCloudPlatform/cloud-run-button)?

**Comment by ahmetb** [comment]
Yeah that’s doable, although since we expect almost everyone to turn the knobs, we are not investing in it right now.

**Comment by ahmetb** [comment]
Actually one thing that won’t work is deploying Cloud Run apps with a custom Service account. The button does not support that right now.
